app/api.py

```diff
@@ -359,28 +359,50 @@ def create_ticket(ticket: TicketCreate) -> Ticket:
         return _create_ticket(ticket)
 
 
-def _get_ticket(status: TicketStatus | None, type_: IssueType | None):
-    """Get tickets with optional filters."""
-    query = TicketModel.select()
-
-    if status is not None:
-        query = query.where(TicketModel.status == status)
-
-    if type_ is not None:
-        query = query.where(TicketModel.type == type_)
-
-    with db:
-        return list(query.dicts())
-
-
 @api_v1_router.get("/tickets")
-def get_tickets(status: TicketStatus | None = None, type_: IssueType | None = None):
+def get_tickets(
+    status: TicketStatus | None = None,
+    type_: IssueType | None = None,
+    page: int = 1,
+    page_size: int = 10,
+):
     """Get all tickets.
 
     This function is used to get all tickets with status open.
     """
     with db:
-        return _get_ticket(status, type_)
+        offset = (page - 1) * page_size
+        # Get the total number of opend and type tickets
+        count = (
+            TicketModel.select()
+            .where(
+                (TicketModel.status == status if status else True)
+                & (TicketModel.type == type_ if type_ else True)
+            )
+            .count()
+        )
+        max_page = count // page_size + int(count % page_size != 0)
+        if max_page * page_size < count:
```

Comment:
You don't need this anymore!
It was the idea behind `int(count % page_size != 0)`: if count is not a multiple of page size, we add an extra page.